python/Ganga/Runtime/Repository_runtime.py

```diff
@@ -134,7 +134,15 @@ def bootstrap():
         registry.startup()
         logger.debug("started " + registry.info(full=False))
         if registry.name == "prep":
-            registry.print_other_sessions()
+            # Get a list of strings describing other running sessions
+            other_sessions = registry.get_other_sessions()
```

Comment:
If this was
```python
other_sessions = registry.repository.get_other_sessions()
```
then it would remove the need for the function you've had to add to the registry.